rs/tests/test_canisters/message/src/main.rs

```diff
@@ -18,12 +15,7 @@ fn read() -> Option<String> {
     MSG.with(|msg| (*msg.borrow()).clone())
 }
 
-#[export_name = "canister_update forward"]
-fn forward_() {
-    over_async_may_reject(candid_one, forward)
-}
-
-#[candid_method(update, rename = "forward")]
+#[ic_cdk_macros::update]
```

Comment:
NCR; 

Is it preferred to import / qualify this macro from `ic_cdk_macros` or `ic_cdk`? They seem to be the same thing, and both occur roughly the same number of times in this repo.

Reply:
I do not know. I preserved the style used in this file at least.